plugins/queue/rabbitmq_amqplib.js

```diff
@@ -39,7 +40,7 @@ exports.init_amqp_connection = function () {
     var autoDelete = cfg.autoDelete === "true" || false;
     deliveryMode = cfg.deliveryMode || 2;
 
-    amqp.connect("amqp://"+user+":"+password+"@"+host+":"+port, function (err, conn){
+    amqp.connect("amqp://"+user+":"+password+"@"+host+":"+port+vhost, function (err, conn){
```

Comment:
Yes, my sugestion is to include it in config file as part of the variable. Like "vhost=/haraka"